modules/nlu/src/backend/index.ts

```diff
@@ -37,13 +38,13 @@ const onBotMount = async (bp: typeof sdk, botId: string) => {
   if (!app) {
     throw new AppNotInitializedError()
   }
-
+  const moduleConfig: Config = await bp.config.getModuleConfig('nlu')
   const botConfig = await bp.bots.getBotById(botId)
   if (!botConfig) {
     throw new Error(`No config found for bot ${botId}`)
   }
 
-  await app.mountBot(botConfig)
+  await app.mountBot(botConfig, moduleConfig.queueTrainingOnBotMount)
```

Comment:
Hey @charlescatta, why is this config not passed at ctor when bootstrapping?

Reply:
Both works, but lets put all the parameters at the same place! We'll change it if we see a use case